pkgs/development/python-modules/import-expression/default.nix

```diff
@@ -35,6 +35,7 @@ buildPythonPackage rec {
       mit
       psfl
     ];
+    broken = pythonAtLeast "3.12";
```

Comment:
Can you link to an upstream issue?

Reply:
No, but this is caused by a failed test when building for 3.12
![image](https://github.com/NixOS/nixpkgs/assets/82718618/4b0b809c-a94d-4bbf-a829-df35a7004c58)